pkg/vpnc-scripts-tarball

```diff
@@ -0,0 +1,16 @@
+[mirrors]
+https://github.com/alonbl/vpnc-scripts/archive/master.zip
```

Comment:
why .zip ?
also you shouldn't use master but a specific commit if you dont wanna use a release tag for some reason.
master is a moving target and as soon as upstream pushes a commit, the hash etc won't match anymore